modules/serverless-rn-1-15-0.adoc

```diff
@@ -0,0 +1,25 @@
+// Module included in the following assemblies:
+//
+// * serverless/release-notes.adoc
+
+[id="serverless-rn-1-15-0_{context}"]
+= Release Notes for Red Hat {ServerlessProductName} 1.15.0
+
+[id="new-features-1.15.0_{context}"]
+== New features
+
+* {ServerlessProductName} now uses Knative Serving 0.21.0.
+* {ServerlessProductName} now uses Knative Eventing 0.21.0.
+* {ServerlessProductName} now uses Kourier 0.21.0.
+* {ServerlessProductName} now uses Knative `kn` CLI 0.21.0.
+* {ServerlessProductName} now uses Knative Kafka 0.21.0.
```

Comment:
we are actually using `0.21.1` version: https://github.com/openshift-knative/serverless-operator/blob/release-1.15/olm-catalog/serverless-operator/manifests/serverless-operator.clusterserviceversion.yaml#L551